prow/plugins/plugins.go

```diff
@@ -157,17 +157,17 @@ type Configuration struct {
 	Owners Owners `json:"owners,omitempty"`
 
 	// Built-in plugins specific configuration.
-	Triggers      []Trigger     `json:"triggers,omitempty"`
-	Heart         Heart         `json:"heart,omitempty"`
-	Milestone     Milestone     `json:"milestone,omitempty"`
-	Slack         Slack         `json:"slack,omitempty"`
-	ConfigUpdater ConfigUpdater `json:"config_updater,omitempty"`
-	Blockades     []Blockade    `json:"blockades,omitempty"`
-	Approve       []Approve     `json:"approve,omitempty"`
-	Blunderbuss   Blunderbuss   `json:"blunderbuss,omitempty"`
-	RequireSIG    RequireSIG    `json:"requiresig,omitempty"`
-	SigMention    SigMention    `json:"sigmention,omitempty"`
-	Cat           Cat           `json:"cat,omitempty"`
+	Triggers      []Trigger            `json:"triggers,omitempty"`
+	Heart         Heart                `json:"heart,omitempty"`
+	RepoMilestone map[string]Milestone `json:"repo_milestone,omitempty"`
```

Comment:
The shape of this seems different compared to other plugin configs that allow per-repo configuration, ie: why not a `[]Milestone`?

Reply:
Would be pretty inefficient? in essence this one is more similar to the "ExternalPlugins" configuration.